experimental/web_dashboard/lib/src/app.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'pages/home.dart';
+import 'api/api.dart';
+import 'api/mock.dart';
+
+/// An app that shows a responsive dashboard.
+class DashboardApp extends StatefulWidget {
+  @override
+  _DashboardAppState createState() => _DashboardAppState();
+}
+
+class _DashboardAppState extends State<DashboardApp> {
```

Comment:
Right now, it doesn't look like this needs to be Stateful. If that's just future-proofing for changes you've already got in mind, though, carry on. :)

Reply:
fixed